starwars/sets/TOR.txt

```diff
@@ -81,7 +81,7 @@ Resource Shortage	TOR	TOR025_Resource_Shortage	N	Mission		X				R	25		Choose one:
 Revan (B)	TOR	TOR026_Revan_B	L	Character	Old Republic Jedi Master	10	70	7	7	R	26		Overkill | Pay 7 Force -> Deflect 4 | Pay 2 Force -> Evade 3	
 Revan (C)	TOR	TOR027_Revan_C	L	Character	Old Republic Jedi Master	7	50	6	6	R	27		Foresight: You may remove a Mission card with printed build cost X from your discard pile from the game. If you do, look at the top 4 cards of your deck and put up to X of them into your hand. Put the rest on the top of your deck in any order. | Pay 3 Force -> Evade 3	
 Revan (D)	TOR	TOR105_Revan_D	L	Character	Old Republic Jedi Master	6	50	5	5	C	105		Pay 2 Force -> Revan gets +1 power and Critical Hit 1 for this attack. | Pay 2 Force -> Evade 2	
-Revan's Mask	TOR	TOR028_Revan_s_Mask_A	N	Equipment	Character - Armor Defense	1				R	28		Pay 1 build point, 1 Force -> Equip | Equipped Jedi or Sith gets +1 health | Equipped Darth Revan or Revan also gets "When the battle phase starts, you may choose an ability on another card in Revan's stack. If you do, Revan gets the chosen ability until end of turn."	
+Revan's Mask (A)	TOR	TOR028_Revan_s_Mask_A	N	Equipment	Character - Armor Defense	1				R	28		Pay 1 build point, 1 Force -> Equip | Equipped Jedi or Sith gets +1 health | Equipped Darth Revan or Revan also gets "When the battle phase starts, you may choose an ability on another card in Revan's stack. If you do, Revan gets the chosen ability until end of turn."	
```

Comment:
this will stop Revan's Mask loading from people's decks. Can you take this one out?

Reply:
I've reverted the Revan's Mask fix and moved it to its own pull request (#50). This pull request now fixes only issues #30 #31 #32 #33